aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -47,6 +47,19 @@ module aptos_token::token {
     const EWITHDRAW_ZERO: u64 = 22;
     const ENOT_TRACKING_SUPPLY: u64 = 23;
     const ENFT_NOT_SPLITABLE: u64 = 24;
+    const ECOLLECTION_NAME_TOO_LONG: u64 = 25;
+    const ENFT_NAME_TOO_LONG: u64 = 26;
+    const EURI_TOO_LONG: u64 = 27;
+
+    //
+    // Constants
+    //
+
+    const MAX_COLLECTION_NAME_LENGTH: u64 = 32;
+    const MAX_NFT_NAME_LENGTH: u64 = 32;
+    // URI lengths: Mean: 76.97, StdDev: 37.41, 95th%: 157, 99th%: 199 (http://www.supermind.org/blog/740/average-length-of-a-url-part-2)
+    const MAX_URI_LENGTH: u64 = 200;
```

Comment:
Why not 2048?  Those S3 and ipfs links can get long.  I don't think we want to be too restrictive here since it's not really querying by index

Reply:
not 2048 long though; You okay with 256? 512?